### DISCUSSION TITLE
JEST interaction

### DISCUSSION THREAD

**Discussion by M1r0H** [Papooch/nestjs-cls#87] [Q&A]

**Comment by Papooch** [comment]
Hi, I'm not sure if you edited the question to be empty in error, but regarding this:
> Hi all, working with CLS as part of NEST js. Need help or advice on how CLS interacts with JEST? And does it interact at all? Because when writing tests, I encountered athe loss of my storage.

Please do have look at these issues:
* https://github.com/Papooch/nestjs-cls/issues/49
* https://github.com/Papooch/nestjs-cls/issues/60

Maybe one of them is related to your problem. If not, please share as much as possible about the problem you're experiencing (ideally a minimal reproduction)
